### PR TITLE
feat: update outdated packages

### PR DESCRIPTION
**Pull request by harish-chandra-89**
Update node version 8 to 12, dependencies version updated and fixed vulnerabilities

### CODE REVIEW

File: bin/cmd.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env node
 const opts = require('../options')
 
-if (process.version.match(/v(\d+)\./)[1] < 6) {
+if (process.version.match(/v(\d+)\./)[1] < 12) {
   // eslint-disable-next-line no-console
-  console.error('standard: Node 6 or greater is required. `standard` did not run.')
+  console.error('standard: Node 12 or greater is required. `standard` did not run.')
```
Comment (mattcan) [inline bin/cmd.js]: Let's bump from 6 -> 8 instead.

File: bin/cmd.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env node
 const opts = require('../options')
 
-if (process.version.match(/v(\d+)\./)[1] < 6) {
+if (process.version.match(/v(\d+)\./)[1] < 12) {
```
Comment (mattcan) [inline bin/cmd.js]: Let's bump from 6 -> 8 instead.

File: package.json [hunk 3]
```diff
@@ -17,7 +17,7 @@
     ""
   ],
   "engines": {
-    "node": ">=8"
+    "node": ">=12"
```
Comment (mattcan) [inline package.json]: Please revert to 8

File: bin/cmd.js (hunk 1 above)
Comment (harish-chandra-89) [reply bin/cmd.js]: @mattcan Changes done

File: bin/cmd.js (hunk 2 above)
Comment (harish-chandra-89) [reply bin/cmd.js]: @mattcan Changes done

File: package.json (hunk 3 above)
Comment (harish-chandra-89) [reply package.json]: @mattcan Changes done

**Pull request merged**